src/styles/blocks/container.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+  margin: 0 auto;
+  width: $container-width;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px 0;
+
+  &__card {
+    margin-right: 48px;
```

Comment:
The margin-right property may not push the fourth card to the next line if the width of the container is not enough. It's recommended to use gap property with grid instead of margin-right.